src/App.tsx

```diff
@@ -30,26 +93,44 @@ export const App: React.FC = () => {
 
         <section className="todoapp__main">
           {/* This is a completed todo */}
-          <div className="todo completed">
-            <label className="todo__status-label">
-              <input
-                type="checkbox"
-                className="todo__status"
-                checked
-              />
-            </label>
-
-            <span className="todo__title">Completed Todo</span>
-
-            {/* Remove button appears only on hover */}
-            <button type="button" className="todo__remove">×</button>
-
-            {/* overlay will cover the todo while it is being updated */}
-            <div className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
+          {todos.map((todo) => (
+            <div
+              key={todo.id}
+              className={classNames('todo', { completed: todo.completed })}
+              role="button"
+              tabIndex={0}
+              onClick={() => setActiveTodoId(todo.id)}
+              onKeyDown={(e) => {
+                if (e.key === 'Enter' || e.key === ' ') {
+                  setActiveTodoId(todo.id);
+                }
+              }}
+            >
+              <label className="todo__status-label">
+                <input
+                  type="checkbox"
+                  className="todo__status"
+                  checked={todo.completed}
+                  onChange={() => handleTodoCheck}
```

Comment:
it wouldn't work at all
```suggestion
                  onChange={handleTodoCheck}
```